locations/spiders/poste_italiane_it.py

```diff
@@ -0,0 +1,84 @@
+import re
+
+from scrapy.http import JsonRequest
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.hours import CLOSED_IT, DAYS_IT, DELIMITERS_IT, NAMED_DAY_RANGES_IT, NAMED_TIMES_IT, OpeningHours
+from locations.json_blob_spider import JSONBlobSpider
+
+
+class PosteItalianeITSpider(JSONBlobSpider):
+    name = "poste_italiane_it"
+    # web page: https://www.poste.it/cerca/index.html#/vieni-in-poste
+    start_urls = ["https://mapcollection.poste.it/v2/map/geoList"]
+    locations_key = ["data", "listaPunti"]
+    POSTE_BRAND = {"brand": "Poste Italiane", "brand_wikidata": "Q495026"}
+    PUNTO_POSTE_BRAND = {
+        "post_office:brand": "Poste Italiane",
+        "post_office:brand:wikidata": "Q495026",
+    }
+    TOBACCO_MATCH = re.compile(r"^Tabacc.+\s*N\.\s*(\d+)$")
+
+    def start_requests(self):
+        for url in self.start_urls:
+            yield JsonRequest(
+                url,
+                data={
+                    "lon": 12.480178129359274,
+                    "lat": 41.89637400808064,
+                    "spanLon": 7,
+                    "spanLat": 7,
+                    "tipoPunto": ["UfficioPostale", "PuntoPoste", "PuntoPosteLocker"],
```

Comment:
FYI there is also:
```
      5 MerchantVirtuale
      6 SpazioFilatelico
  38982 Merchant
```
Don't know if they are interesting.

Reply:
Thanks, didn't notice them - I found also CassettaPostale that is also interesting
Merchants would only be tagged "anything=yes", as they contain fuel pumps, shops, amenities, healthcare facilities.... anything that accepts quick payments using Poste app
In #11670 I added post boxes and philately shops